homeassistant/components/victron_remote_monitoring/strings.json

```
@@ -0,0 +1,92 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Enter your VRM API token. We will then fetch your available sites.",
+        "data": {
+          "api_key": "VRM API Key"
```

This needs sentence-casing, but we should use the common string here instead anyhow:

```suggestion
          "api_key": "[%key:common::config_flow::data::api_key%]"
```

"VRM" is already in the descriptions above and below.